spec/jira_api_spec.rb

```diff
@@ -0,0 +1,28 @@
+# -- coding: utf-8
```

Comment:
oh.. but that magic comment was unneeded, so remove it is no problem for now.
Thank you for your information!